vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -708,4 +717,14 @@ private void attachComponentRenderer() {
         setDimension(ElementConstants.STYLE_MIN_HEIGHT, minHeight);
         setDimension(ElementConstants.STYLE_MAX_HEIGHT, maxHeight);
     }
+
+    /**
+     * @throws UnsupportedOperationException
+     *             Adding styles to dialog overlay are not supported
+     */
+    @Override
+    public Style getStyle() {
+        throw new UnsupportedOperationException();
```

Comment:
Let's add the description here too:

```suggestion
        throw new UnsupportedOperationException(
                "Dialog does not support adding styles to overlay");
```